build.gradle

```diff
@@ -285,9 +285,16 @@ project(':iceberg-aliyun') {
     implementation project(':iceberg-common')
 
     compileOnly 'com.aliyun.oss:aliyun-sdk-oss'
```

Comment:
If we plan to add this into bundled `aliyun-iceberg` jar,  then this line should use `implementation` rather than `compileOnly`.  Because we only add classes from `implementation` jars when packaging the whole iceberg jar.

Reply:
I think service dependency jars are not added to the bundled jar to minimize bundled jar size.